runtime/compiler/optimizer/IdiomTransformations.cpp

```diff
@@ -1641,13 +1641,13 @@ CISCTransform2FindBytes(TR_CISCTransformer *trans)
 
    TR::Node *alenNode   = TR::Node::create( baseRepNode, TR::arraylength, 1);
    alenNode->setAndIncChild(0, baseNode);
-   ////findBytesNode->setSymbolReference(comp->getSymRefTab()->findOrCreateFindBytesSymbol());
+   /*findBytesNode->setSymbolReference(comp->getSymRefTab()->findOrCreateFindBytesSymbol());*/
```

Comment:
The source code for the JIT compiler doesn't usually use the block form of comment for a single line of comments in the code. May I ask you to change this to the following?

```
//findBytesNode->setSymbolReference(comp->getSymRefTab()->findOrCreateFindBytesSymbol());
```